cocotb_build_libs.py

```diff
@@ -633,10 +633,10 @@ def get_ext():
     )
     ext.append(modelsim_vhpi_ext)
 
-    vsim_path = find_executable("vdbg")
+    vsim_path = find_executable("vopt")
     if vsim_path is None:
         logger.warning(
-            "Modelsim/Questa executable (vdbg) executable not found. No FLI interface will be available."
+            "Modelsim/Questa executable (vopt) executable not found. No FLI interface will be available."
```

Comment:
Maybe?
```suggestion
            "Modelsim/Questa executable (vopt) executable not found. No FLI interface will be available because we cannot find the installation directory."
```

Reply:
I think this reads a little better.
```suggestion
            "Modelsim/Questa installation not found: did not find the 'vopt' executable. No FLI interface will be available."
```

We really should highlight this in yellow or something.

Reply:
Yes, Your is better.